concepticondata/conceptlists.tsv

```diff
@@ -270,3 +270,4 @@ Kleinewillinghoefer-2015-119	Kleinewillinghoefer, Ulrich	2015		119	basic	English
 Othaniel-2017-300	Othaniel, Nlabephee Kefas	2017		300	basic	English	Jen languages		Othaniel2017		This list is taken from the [unpublished master thesis](:bib:Othaniel2017), presenting a phonological comparative study of the Jen language cluster, and it bears strong resemblance to the [200-item list by Swadesh (1952)](Swadesh-1952-200).		
 Nordlinger-2019-57	Nordlinger, Rachel and Thieberger, Nick	2019		57	basic	English	Australian	https://arts.unimelb.edu.au/research-unit-for-indigenous-language/research/current-research-projects/50-words-project	Nordlinger2019		The given list underlies the [50 words project](https://arts.unimelb.edu.au/research-unit-for-indigenous-language/research/current-research-projects/50-words-project), initiated by the Research Unit for Indigenous Language and aimed at collecting data on Australian languages for, primarily, teaching purposes. Note, however, that, despite the name of the project, the list represents 57 lexical items.		
 Ritchie-2019-111	Ritchie, Sandy and Sproat, Richard and Gorman, Kyle and van Esch, Daan and Schallhart, Christian and Bampounis, Nikos and Brard, Benoît and Fromseier Mortensen, Jonas and Holt, Millie and Mahon, Eoin	2019		111	questionnaire	English	Global	https://www.isca-speech.org/archive/Interspeech_2019/pdfs/2807.pdf	Ritchie2019		This list represents 111 numerals, used in the development of computational solutions to number verbalization in various languages. This initiative is part of a broader research effort at Google, enhancing the available machine learning techniques for building number names grammars across a wide range of languages for application in automatic speech recognition and text-to-speech synthesis systems.	3530-3534	
+Joo-2019-100	Joo, Ian	2019		100	ranked, basic	English	Global		Joo2019		This list of 100-item, accurately reflecting the [Leipzig-Jakarta concept list](:ref:Tadmor-2009-100), was employed in the study of phonosemantic patterns in the basic vocabulary of the 66 genealogically unrelated languages.	1-12	
```

Comment:
Some copy-editing: This list of 100 item**s** is based on the ... and was used in the study of phonosemantic patterns in the basic vocabulary of ~~the~~ 66 genealogically unrelated languages